api/lib/libDependencies.js

```diff
@@ -0,0 +1,60 @@
+const { doJigsawGetRequest, jigsawEnv } = require('../lib/JigsawUtils');
+const SqlServerConnection = require('./SqlServerConnection');
+const jigsawSearchGateway = require('./gateways/Jigsaw/JigsawSearchGateway')({
+  doJigsawGetRequest,
+  jigsawEnv,
+  buildSearchRecord: require('./entities/SearchRecord')()
+});
+const academyBenefitsSearchGateway = require('./gateways/Academy-Benefits/AcademyBenefitsSearchGateway')(
+  {
+    db: new SqlServerConnection({
+      dbUrl: process.env.ACADEMY_DB
+    }),
+    buildSearchRecord: require('./entities/SearchRecord')()
```

Comment:
All these buildSearchRecord instantiations can happen once at the top of the file